2022-01-19 11:01:46,178 INFO    MainThread:30836 [wandb_setup.py:_flush():71] setting env: {}
2022-01-19 11:01:46,178 INFO    MainThread:30836 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-19 11:01:46,178 INFO    MainThread:30836 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220119_110146-14gna859\logs\debug.log
2022-01-19 11:01:46,178 INFO    MainThread:30836 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220119_110146-14gna859\logs\debug-internal.log
2022-01-19 11:01:46,178 INFO    MainThread:30836 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000230960D76A0>
2022-01-19 11:01:46,193 INFO    MainThread:30836 [wandb_init.py:init():404] calling init triggers
2022-01-19 11:01:46,193 INFO    MainThread:30836 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-19 11:01:46,193 INFO    MainThread:30836 [wandb_init.py:init():449] starting backend
2022-01-19 11:01:46,193 INFO    MainThread:30836 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-19 11:01:46,193 INFO    MainThread:30836 [backend.py:ensure_launched():199] starting backend process...
2022-01-19 11:01:46,600 INFO    MainThread:30836 [backend.py:ensure_launched():204] started backend process with pid: 14384
2022-01-19 11:01:46,600 INFO    MainThread:30836 [wandb_init.py:init():458] backend started and connected
2022-01-19 11:01:46,615 INFO    MainThread:30836 [wandb_run.py:_label_probe_notebook():917] probe notebook
2022-01-19 11:01:46,615 INFO    MainThread:30836 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-19 11:01:46,615 INFO    MainThread:30836 [wandb_init.py:init():519] updated telemetry
2022-01-19 11:01:46,615 INFO    MainThread:30836 [wandb_init.py:init():549] communicating current version
2022-01-19 11:01:48,272 INFO    MainThread:30836 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-19 11:01:48,272 INFO    MainThread:30836 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-19 11:01:48,443 INFO    MainThread:30836 [wandb_init.py:init():592] starting run threads in backend
2022-01-19 11:01:49,441 INFO    MainThread:30836 [wandb_run.py:_console_start():1816] atexit reg
2022-01-19 11:01:49,441 INFO    MainThread:30836 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-19 11:01:49,441 INFO    MainThread:30836 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-19 11:01:49,441 INFO    MainThread:30836 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-19 11:01:49,441 INFO    MainThread:30836 [wandb_init.py:init():619] run started, returning control to user process
2022-01-19 11:01:51,888 INFO    MainThread:30836 [wandb_run.py:finish():1463] finishing run austinbergstrom/places_dry_run/14gna859
2022-01-19 11:01:51,966 INFO    MainThread:30836 [jupyter.py:save_history():429] not saving jupyter history
2022-01-19 11:01:51,966 INFO    MainThread:30836 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-19 11:01:51,966 INFO    MainThread:30836 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-01-19 11:01:51,966 INFO    MainThread:30836 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-01-19 11:01:51,966 INFO    MainThread:30836 [wandb_run.py:_restore():1758] restore
2022-01-19 11:01:52,904 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-19 11:01:53,493 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-19 11:01:53,634 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-19 11:01:54,511 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4455
}

2022-01-19 11:01:54,621 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4481
}

2022-01-19 11:01:54,730 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4481
  total_bytes: 4481
}

2022-01-19 11:01:54,840 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4481
  total_bytes: 4481
}

2022-01-19 11:01:54,949 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4481
  total_bytes: 4481
}

2022-01-19 11:01:55,058 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4481
  total_bytes: 4481
}

2022-01-19 11:01:55,168 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4481
  total_bytes: 4481
}

2022-01-19 11:01:55,527 INFO    MainThread:30836 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4481
  total_bytes: 4481
}
local_info {
}

2022-01-19 11:01:56,683 INFO    MainThread:30836 [wandb_run.py:_append_files():2184] logging synced files
